modules/using-cluster-compare-telco-ref.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+
+// *scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-crs.adoc
+// *scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-du-crs.adoc
+// *scalability_and_performance/cluster-compare/using-the-cluster-compare-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+ifeval::["{context}" == "using-cluster-compare-plugin"]
+:cluster-compare:
+endif::[]
+
+ifeval::["{context}" == "ran-core-ref-design-crs"]
+:cluster-compare-core:
+endif::[]
+
+ifeval::["{context}" == "ran-ref-design-crs"]
+:cluster-compare-ran:
+endif::[]
+
+[id="using-cluster-compare-telco_ref_{context}"]
+ifdef::cluster-compare[= Example: Comparing a cluster with the telco core reference configuration]
+ifdef::cluster-compare-core,cluster-compare-ran[= Comparing a cluster with the {rds} reference configuration]
+
+//Intro for procedure in telco core/RAN RDS assembly
+ifdef::cluster-compare-core,cluster-compare-ran[]
+After you deploy a {rds} cluster, you can use the `cluster-compare` plugin to assess the cluster's compliance with the {rds} reference design specification (RDS). The `cluster-compare` plugin is an OpenShift CLI (`oc`) plugin. The plugin uses a {rds} reference configuration to validate the cluster with the {rds} custom resources (CRs). 
+
+The plugin-specific reference configuration for {rds} is packaged in a container image with the {rds} CRs. 
+
+For further information about the `cluster-compare` plugin, see "Understanding the cluster-compare plugin" in the "Additional resources" section.
```

Comment:
Per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations), references like these don't need to reference "Additional resources" by name:
```suggestion
For further information about the `cluster-compare` plugin, see "Understanding the cluster-compare plugin".
```
Check for other instances like these.